lib/ansible/modules/network/iosxr/iosxr_config.py

```diff
@@ -26,7 +26,7 @@
 module: iosxr_config
 version_added: "2.1"
 author: "Ricardo Carrillo Cruz (@rcarrillocruz)"
-short_description: Manage Cisco IOS XR configuration sections
+short_description: Manage Cisco IOS-XR configuration sections
```

Comment:
I'm not super sure, but everything I've seen seems to indicate the name is not actually hyphenated